src/aat/resources/features/F-000/S-000.td.json

```diff
@@ -27,7 +27,7 @@
     "responseMessage": "OK",
     "headers" : {
       "Content-Length" : "[[ANY]]",
-      "Content-Type" : "application/vnd.spring-boot.actuator.v2+json;charset=UTF-8",
+      "Content-Type" : "application/vnd.spring-boot.actuator.v3+json;",
```

Comment:
Deprecate MediaType.APPLICATION_JSON_UTF8 in favor of APPLICATION_JSON from Spring 5.2
spring-projects/spring-framework#22788